File: python/leetcode/transpose-matrix.py
class Solution:
    def transpose(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        B = []
        for i in range(len(A[0])):
            for j in (range(len(A))):
                if len(B) <= i:
                    B.append([]);
                B[i].append(A[j][i]);
        return B

File: javascript/leetcode/transpose-matrix.js
/**
 * @param {number[][]} A
 * @return {number[][]}
 */
var transpose = function(A) {
    B = [];
    for (let i = 0; i < A[0].length; i++) {
        for (let j = 0; j < A.length; j++) {
            if (B.length <= i) {
                B.push([]);
            }
            B[i].push(A[j][i]);
        }
    }
    return B;
};
